source/index.html.md

```diff
@@ -857,18 +2159,215 @@ var options = {
 	filters: {
 		$title: {
 			$contains: 'adventure'
+		},
+		$timestampUpdated: {
```

Comment:
What do you think about adding just a simple equality match. So we would have three filter examples

1. Meta keyword like `$title` with an operator like `$contains` ✅ 
2. Content value with an operator like `$greaterThan` (need to change `$timestampUpdated`)
3. Content value with simple equality match (need to add)

With those examples I _think_ it would be clear how you use the different filter operators and filter keys. Just and idea though